src/Data/Interpolation.hs

```diff
@@ -184,24 +184,20 @@ instance (ToTemplateValue a, ToJSON a) => ToJSON (Uninterpolated a) where
     Templated x -> toJSON x
     Literal x -> toJSON x
 
-maybeGen :: Gen a -> Gen (Maybe a)
-maybeGen x = frequency [(1, pure Nothing), (3, Just <$> x)]
-
-noEnv, varNameAllowed :: Gen T.Text
-noEnv = fmap T.pack $ arbitrary `suchThat` (\ s -> not ("_env:" `isPrefixOf` s) && not (null s))
-varNameAllowed = fmap T.pack . listOf1 $ arbitrary `suchThat` (\c -> isAlphaNum c || c == '_')
-
 instance Arbitrary TemplateKey where
   arbitrary = TemplateKey <$> varNameAllowed
+    where varNameAllowed = fmap T.pack . listOf1 $ arbitrary `suchThat` (\c -> isAlphaNum c || c == '_')
 
-instance {-# OVERLAPPABLE #-} Arbitrary a => Arbitrary (Uninterpolated a) where
-  arbitrary = oneof
-    [ Literal <$> arbitrary
-    , Templated <$> (Template <$> arbitrary <*> arbitrary)
+instance Arbitrary1 Uninterpolated where
+  liftArbitrary g = oneof
+    [ Literal <$> g
+    , Templated <$> (Template <$> arbitrary <*> liftArbitrary g)
     ]
 
+instance {-# OVERLAPPABLE #-} Arbitrary a => Arbitrary (Uninterpolated a) where
+  arbitrary = arbitrary1
+
 instance {-# OVERLAPPING #-} Arbitrary (Uninterpolated T.Text) where
-  arbitrary = oneof
-    [ Literal <$> noEnv
-    , Templated <$> (Template <$> arbitrary <*> maybeGen noEnv)
-    ]
+  arbitrary = liftArbitrary noEnv
+    where noEnv = fmap T.pack $ arbitrary `suchThat` (\ s -> not ("_env:" `isPrefixOf` s) && not (null s))
+
```

Comment:
👮Extra newline police 👮

Reply:
I'm sorry officer. I updated my editor configuration so it will never happen again.